app/helpers/comments_helper.rb

```diff
@@ -1,13 +1,14 @@
 module CommentsHelper
-	def comments_json(commentable_object)
-		if commentable_object
-			eval(ActiveModel::ArraySerializer.new(commentable_object.comments.recent_last, each_serializer: CommentSerializer).to_json)
-		end
-	end
+  def comments_json(commentable_object)
+    if commentable_object
+      eval(ActiveModel::ArraySerializer.new(commentable_object.comments.recent_last,
+                                            each_serializer: CommentSerializer).to_json)
+    end
+  end
 
-	def current_user_json
-		if current_user
-			eval(UserSerializer.new(current_user).to_json)
-		end
-	end
+  def current_user_json(user = current_user)
```

Comment:
@rathanbeyond that is a very good question and I agree with you.  Basically I just added that parameter to the method so that 1. you don't have to change anything that already exists but also 2. you can pass in a param if you want to.

Basically the reason is that it makes it easier to test helpers; I was testing the helper in the console and often you may want to write helper specs.  If your helper has to have some kind of external infrastructure built up before it can work (or be tested), that makes your tests kind of bloated.  So a good way to make easily testable helpers (or methods) is to inject the dependencies via parameter, but also have default values so that when they are used in the real world, you don't need to clutter the call.  Does that make sense?